src/main.go

```diff
@@ -116,6 +119,20 @@ func (r *RadiusCredentials) StartGCWorker() {
 	go r.gcworker()
 }
 
+func (r *RadiusCredentials) accountingCron() *cron.Cron {
+        // hourly accounting cron job
+     	c := cron.New()
+	c.AddFunc("0 0 * * * *", func() {
+		// accounting
+		err := r.accounting("/var/log/ganted/access.log", "/var/log/ganted/archive.log")
```

Comment:
Shouldn't this use file paths from environment instead of hard-coded paths?

Reply:
Fixed, use environment variable GANTED_LOG_DIR, but force the access.log, error.log, archive.log are under GANTED_LOG_DIR.